.github/workflows/test_fe_dagger.yaml

```diff
@@ -0,0 +1,44 @@
+name: Test FE Dagger
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+env:
+  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+
+jobs:
+  changes:
+    runs-on: ubuntu-latest
+    outputs:
+      frontend: ${{ steps.filter.outputs.frontend }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dorny/paths-filter@v3
+        id: filter
+        with:
+          filters: |
+            frontend:
+              - 'frontend/**'
+              - 'lsp/**'
+              - 'openapi/**'
```

Comment:
could we add `dagger/**` to this list? looks like this workflow didnt run since nothing in frontend changed

Reply:
Yes good call!